### PR TITLE
policy for functions and classes

### PR DESCRIPTION
**Pull request by MelissaGraham**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -113,10 +113,14 @@ It should have a markdown cell that provides explanations and/or links to extern
...
-``1.2.1 Define global cosmological parameter values`` or ``1.2.2 Define a function to make an image cutout``.
-If applicable, it is preferred to leave this subsection header in the document and state that no additional functions and parameters are used.
+Globally defined utility functions, classes, plotting defaults, or constants should be here.
+
+If a notebook has no functions or parameters to define, it is preferred to leave this subsection header in the document
+and state that no additional functions and parameters are used.
+
+If a notebook has many such things to define, it is acceptable to rename the subsection to be more specific to the notebook,
+and/or to use sub-subsections like ``1.2.1 Define global cosmological parameter values`` or ``1.2.2 Define a function to make an image cutout``.
+
```
Comment (plazas) [inline index.rst:123]: At this point, should we warn the reader about the exception below (e.g., "see section "function and classes" below")

File: index.rst [hunk 2]
```diff
@@ -191,6 +195,22 @@ Since use of "extent" is necessary for displaying a WCS overlay for ``deepCoadds
...
+defined in section ``1.2 Define Functions and Parameters``, single-use functions or classes 
+can be defined immediately before they are used.
+
+Functions or classes that are particularly long blocks of code (e.g., >20 lines) can be hidden by going to
+the "View" menu item and choosing "Collapse Selected Code", or by clicking on the blue bar that
+appears to the left of a selected cell.
+Hidden cells should be described in the preceding markdown cell with text like 
+"the following hidden cell contains code that defines the ``make_cmd_plot`` function".
+The first hidden cell in a notebook should include instructions for displaying the cell, such as
+"to see the contents of the hidden cell, select View from the menu bar and then Expand Selected Code".
```
Comment (plazas) [inline index.rst:211]: We could also explain that there are direct ways of expanding the cell with something like: "To see the contents of the hidden cell, select View from the menu bar and then Expand Selected Code, or click on the vertical next to the cell or on the three dots that denote that the cell is hidden."

**Review by plazas — CHANGES_REQUESTED** [review]

File: index.rst (hunk 1 above)
Comment (MelissaGraham) [reply index.rst:124]: implemented, thank you!

File: index.rst (hunk 2 above)
Comment (MelissaGraham) [reply index.rst]: implemented verbatim, thank you!

**Review by plazas — APPROVED** [review]
Looks good, thanks for addressing my comments and making these updates!

**Pull request merged**